utils/build/docker/java/spring-boot/pom.xml

```diff
@@ -32,9 +32,14 @@
             <groupId>org.springframework.session</groupId>
             <artifactId>spring-session-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Do we need this?

Reply:
I've added the logger to get log messages from Spring